src/main/common/utils.js

```diff
@@ -196,10 +197,31 @@ gpii.app.isPointInRect = function (point, rectangle) {
  * In most cases, there's only a single USB drive. But if there's more than one USB drive,
  * then those that do not contain the token file are shown.
  */
-gpii.app.openUSB = function() {
+gpii.app.openUSB = function () {
     gpii.windows.getUserUsbDrives().then(function (paths) {
         fluid.each(paths, function (path) {
             child_process.exec("explorer.exe \"" + path + "\"");
         });
     });
 };
+
+/**
+ * A custom function for handling opening of an .exe file.
+ * @param {String} executablePath - path to executable file
+ */
+gpii.app.launchExecutable = function (executablePath) {
+    var fileProperties = fs.statSync(executablePath);
+
+    // Check that the file is executable
+    if (fileProperties.mode === 33206) {
```

Comment:
Please use octal literal here so it is easier to understand what the required file mode is

Reply:
Done